data_validation/src/spark_rapids_validation_tool/dataproc_wrapper.py

```diff
@@ -79,6 +81,29 @@ def validation(self,
             return
         getattr(validate, check)()
 
+    def validation(self, conf_file: str):
+        with open(conf_file, "r") as file:
+            validate_conf = yaml.safe_load(file)
+        spark_conf = validate_conf['sparkConf']
+        tool_conf = validate_conf['toolConf']
+        self.validation_parse(tool_conf['cluster'],
+                         tool_conf['region'],
+                         tool_conf['check'],
+                         tool_conf['format'],
+                         tool_conf['table1'],
+                         tool_conf['table1_partition'],
```

Comment:
What is the behavior when the arg is not set? Will it be set to "None" by default?

Reply:
According to my debug info, it will be set to `None` which is a `<class 'NoneType'>`. But it will be convert to a "None" string by `argparse` module when I pass it to `metadata_validation.py` or `dataset_validation.py` script